sovrin_client/agent/walleted.py

```diff
@@ -278,6 +278,17 @@ def handleEndpointMessage(self, msg):
         if typ == ACCEPT_INVITE and link is None:
             localIdr = self.wallet.defaultId
         else:
+            # if accept invite is not the message type
```

Comment:
@rajeshkalaria80 yes, as of now if we send ping before accept, it crashes agent, because we need a local identifier and link is accessed inside ping handler, which it does not find since local identifier is not present. Also, yes product team is good with this change.